perception/camera_obstacle_detection/yolo_v3/tensorflow_fp32_coco/execute_model/main.cpp

```diff
@@ -0,0 +1,434 @@
+//
+// Copyright (c) 2020-2021, Arm Limited and Contributors. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <assert.h>
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+#include <vector>
+
+#include <dlpack/dlpack.h>
+#include <tvm/runtime/module.h>
+#include <tvm/runtime/packed_func.h>
+#include <tvm/runtime/registry.h>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+
+// model files
+#define network_dir "./"
+#define network_module_path network_dir "deploy_lib.so"
+#define network_graph_path network_dir "deploy_graph.json"
+#define network_params_path network_dir "deploy_param.params"
+// Name of file containing the human readable names of the classes. One class
+// on each line.
+#define LABEL_FILENAME "labels.txt"
+// Name of file containing the anchor values for the network. Each line is one
+// anchor. each anchor has 2 comma separated floating point values.
+#define ANCHOR_FILENAME "anchors.csv"
+
+// network configurations
+#define TVDTYPE_CODE kDLFloat
+#define TVDTYPE_BITS 32
+#define TVDTYPE_LANES 1
+#define TVDEVICE_TYPE kDLCPU
+#define TVDEVICE_ID 0
+
+// network input dimensions
+#define NETWORK_INPUT_WIDTH 416
+#define NETWORK_INPUT_HEIGHT 416
+#define NETWORK_INPUT_DEPTH 3
+#define NETWORK_INPUT_NAME "input"
+
+// network output dimensions
+#define NETWORK_OUTPUT_HEIGHT_1 13
+#define NETWORK_OUTPUT_WIDTH_1 13
+#define NETWORK_OUTPUT_DEPTH 255
+
+// network output dimensions
+#define NETWORK_OUTPUT_HEIGHT_2 26
+#define NETWORK_OUTPUT_WIDTH_2 26
+
+// network output dimensions
+#define NETWORK_OUTPUT_HEIGHT_3 52
+#define NETWORK_OUTPUT_WIDTH_3 52
+
+// minimum confidence score by which to filter the output detections
+#define SCORE_THRESHOLD 0.5
+
+#define NMS_THRESHOLD 0.45
+
+// filename of the image on which to run the inference
+#define IMAGE_FILENAME "test_image_0.jpg"
+
+// Name of the window to display the detection results
+#define DISPLAY_WINDOW_NAME "YOLO Output"
+
+/// Struct for storing detections in image with confidence scores
+struct BoundingBox
+{
+
```

Comment:
why do we have these extra empty lines? Same question also for lines 151 and 155

Reply:
Done removed the extra empty lines